keyboards/flxlb/zplit/keymaps/via/keymap.c

```diff
@@ -0,0 +1,67 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
}; 
```

Also, setting adjust to 16 means that you're adding 14 empty layers, bloating the firmware size.